recipes/libgpuarray/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: libgpuarray
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/Theano/libgpuarray.git
+  git_tag: v0.6.0
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - git
+    - cmake
+    - python
+    - cython >=0.25
+    - numpy
+    - mako
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - mako
+    - six
+    - nose  # This will be fixed in 0.6.1
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libgpuarray.dylib  # [osx]
+    - test -f ${PREFIX}/lib/libgpuarray.so  # [linux]
+    - if not exists %PREFIX%\\Library\\lib\\libgpuarray.lib exit 1  # [win]
+  imports:
+    - pygpu
+    - pygpu.gpuarray
+
+about:
+  home: http://github.com/Theano/libgpuarray
+  license: BSD
```

Comment:
You're right.  I just dumbly copy and pasted what was in the continuum package.  I'll change it.